### PR TITLE
[automated]: crowdin sync

### PR DESCRIPTION
**Pull request by nodejs-crowdin**
Brian: syncing numerous changes.
I also want to define a policy for when we sync.

- weekly
- as a milestone is hit (language 100% translated and proofread)
- could this be automated via the GH Action

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
> I also want to define a policy for when we sync.

We'll be able to have a cron job that checks if there's a 10% difference on a language unless it's >90%, in which case we'll automatically sync? So with that we w'ill not create pr if there are no "interesting" update/change

**Review by AugustinMauroy — APPROVED** [review]

File: apps/site/pages/fr/about/governance.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Un guide pour les collaborateurs est disponible à l'adresse suivante : [collabo
 
 ## Comité de pilotage technique
 
-Le projet est régi par le [Comité de pilotage technique (TSC)][Technical Steering Committee (TSC)]
+Le projet est régi par le \[Comité de pilotage technique (TSC)]\[]
```
Comment (bmuenzenmeyer) [inline apps/site/pages/fr/about/governance.md]:
```suggestion
Le projet est régi par le [Comité de pilotage technique (TSC)][technical steering committee (tsc)]
```

File: apps/site/pages/id/about/governance.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Panduan untuk Kolaborator disimpan di [collaborator-guide.md][].
 
 ## Komite Pengarah Teknis
 
-Proyek ini diatur oleh [Komite Pengarah Teknis][Technical Steering Committee (TSC)] dalam bahasa inggris Technical Steering Committee (TSC) yang bertanggung jawab atas panduan tingkat tinggi proyek. TSC adalah bagian dari Kolaborator aktif yang dinominasikan oleh anggota TSC lain yang sudah ada.
+Proyek ini diatur oleh \[Komite Pengarah Teknis]\[] dalam bahasa inggris Technical Steering Committee (TSC) yang bertanggung jawab atas panduan tingkat tinggi proyek. TSC adalah bagian dari Kolaborator aktif yang dinominasikan oleh anggota TSC lain yang sudah ada.
```
Comment (bmuenzenmeyer) [inline apps/site/pages/id/about/governance.md]:
```suggestion
Proyek ini diatur oleh [Komite Pengarah Teknis][technical steering committee (tsc)] dalam bahasa inggris Technical Steering Committee (TSC) yang bertanggung jawab atas panduan tingkat tinggi proyek. TSC adalah bagian dari Kolaborator aktif yang dinominasikan oleh anggota TSC lain yang sudah ada.
```

**Review by bmuenzenmeyer — COMMENTED** [review]
i need to pull down the branch to fix apps/site/pages/uk/about/index.mdx - it makes me wonder if something is odd about this markdown, or the parsing, that it keeps getting these links wrong

**Comment by bmuenzenmeyer** [comment]
fixed some lints reported in https://github.com/nodejs/nodejs.org/pull/7081/files and other PRs- will need monitor if it keeps happening

**Review by aymen94 — APPROVED** [review]

**Pull request merged**